.circleci/config.yml

```diff
@@ -68,9 +68,9 @@ jobs:
             command: |
               if [ $IS_BETA == "true" ];
               then
-                npm publish --access public --tag beta
+                npm publish --access public --tag beta latest
```

Comment:
this is temporary where we mark the beta as `latest` since there isn't a GA version. Once we have an GA version this should be removed, since the latest should always be the GA version